Extension/src/githubAPI.ts

```diff
@@ -145,7 +146,18 @@ export async function getTargetBuildInfo(updateChannel: string): Promise<BuildIn
                 return undefined;
             }
 
-            return getTargetBuild(builds, userVersion, updateChannel);
+            const targetBuild: Build | undefined = getTargetBuild(builds, userVersion, updateChannel);
+            if (targetBuild === undefined) {
+                // no oction
+                telemetry.logLanguageServerEvent("NoUPgradeDowngrade");
```

Comment:
I think we should use the same event name for all upgrade responses, but add different "properties" for the action that was taken. This also allows us to log additional information as necessary.

For example:
```
telemetry.logLanguageServerEvent("UpgradeCheck", { "action": "none" })
telemetry.logLanguageServerEvent("UpgradeCheck", { "action": "upgrade", "version": targetBuild.name })
telemetry.logLanguageServerEvent("UpgradeCheck", { "action": "downgrade", "version": targetBuild.name })
```